defaults/main.yml

```diff
@@ -41,10 +46,53 @@ owncloud__base_packages:
   - '{{ [ "libsmbclient" ] if (owncloud__smb_support|bool and owncloud__release | version_compare("9.0", ">=")) else [] }}'
 
 
+# .. envvar:: nextcloud__required_php_packages
+#
+# List of PHP packages required by Nextcloud.
+# Refer to the `official Nextcloud documentation <https://docs.nextcloud.com/server/11/admin_manual/installation/source_installation.html>`__ for details.
+nextcloud__required_php_packages:
```

Comment:
> OK then, let's open this can of worms... Since this breaks the current DebOps Code Standards Policy, what's the reasoning behind this, @ypid?

You are right. Sounded like a good idea at first to make it visible which changes/vars are exclusive to Nextcloud and with the planned rename of the role (some day in the future). I changed it back.

> I wonder how much ownCloud and Nextcloud differ from each other right now and if maintaining the differences between them in one role outweighs the costs of maintaining two separate roles? 

Configuration and interface is basically the same. It clearly makes sense to maintain this in one role.